config/containers/logging/dual-logging.md

```diff
@@ -6,47 +6,57 @@ title: Use docker logs to read container logs for remote logging drivers
 
 ## Overview 
 
-Prior to Docker Engine Enterprise 18.03, the `jsonfile` and `journald` log drivers supported reading 
-container logs using `docker logs`. However, many third party logging drivers had no 
-support for locally reading logs using `docker logs`, including: 
-
-- syslog	
-- gelf	
-- fluentd	
-- awslogs	
-- splunk	
-- etwlogs	
-- gcplogs	
-- Logentries
-
-This created multiple problems, especially with UCP, when attempting to gather log data in an 
-automated and standard way. Log information could only be accessed and viewed through the 
-third-party solution in the format specified by that third-party tool. 
-
-Starting with Docker Engine Enterprise 18.03.1-ee-1, you can use `docker logs` to read container 
-logs regardless of the configured logging driver or plugin. This capability, sometimes referred to 
-as dual logging, allows you to use `docker logs` to read container logs locally in a consistent format, 
-regardless of the remote log driver used, because the engine is configured to log information to the “local” 
-logging driver. Refer to [Configure the default logging driver](/config/containers/logging/configure) for additional information. 
+Prior to Docker Engine Enterprise 18.03, the `jsonfile` and `journald` log
+drivers supported reading container logs using `docker logs`. However, many
+third party logging drivers had no support for locally reading logs using
+`docker logs`, including: 
+
+- syslog
+- gelf
+- fluentd
+- awslogs
+- splunk
+- etwlogs
+- gcplogs
+- logentries
+
+This created multiple problems, especially with UCP, when attempting to gather
+log data in an automated and standard way. Log information could only be accessed
+and viewed through the third-party solution in the format specified by that
+third-party tool. 
+
+Starting with Docker Engine Enterprise 18.03.1-ee-1, you can use `docker logs`
```

Comment:
Should we remove the text `Starting with Docker Engine Enterprise 18.03.1-ee-1,` altogether, or use a Community release which introduced this feature?

Reply:
See my other comment; it's not _yet_ in a Docker Community release, so I need to do a follow-up to prepare for the next release